examples/multi-agent-debate-system/README.md

```diff
@@ -0,0 +1,73 @@
+Debate System Using LLM Agents
+==============================
+
+Overview
+--------
+This project is a debate system powered by LLMs using Langroid, enabling structured debates on various topics such as AI in healthcare, education, intellectual property, and societal biases. The program creates and manages agents that represent opposing sides of a debate, interact with users, and provide constructive feedback based on established debate criteria.
+
+
+Features
+--------
+- Multiple Debate Topics:
+  - AI in Healthcare
+  - AI and Intellectual Property
+  - AI and Societal Biases
+  - AI as an Educator
+- Agent-Based Interaction:
+  - Pro and Con agents for each topic simulate structured debate arguments.
+- Customizable LLM Configuration:
+  - Support for different LLMs like GPT-4, GPT-3.5, and Mistral.
+- Feedback Mechanism:
+  - Provides structured feedback on debate performance based on key criteria.
+- Interactive or Autonomous Mode:
+  - Users can either control interactions manually or let agents autonomously continue debates.
+
+File Structure
+--------------
+- main.py: The entry point of the application. Initializes the system, configures agents, and starts the debate loop.
+- agents.py: Defines functions to create agents for each debate topic, representing the Pro and Con sides.
+- tasks.py: Contains task definitions that encapsulate agent behavior during debates.
+- config.py: Provides functions for configuring global settings and LLM-specific parameters.
+
+Getting Started
+---------------
+Prerequisites
+1. Python 3.8+
+2. Langroid Framework: Install Langroid with all dependencies:
+   pip install "langroid[all]"
```

Comment:
probably don't need "all" ? In fact probably only need `litellm` to be able to switch to claude model.